transifex/native/django/management/utils/push.py

```diff
@@ -167,6 +172,27 @@ def push_strings(self):
         status_code, response_content = tx.push_source_strings(
             self.string_collection.strings.values(), self.purge
         )
+
+        if self.no_wait:
+            Color.echo('Queued')
+            return
+
+        job_url = response_content.get('data').get('links').get('job')
```

Comment:
same as before:

```diff
-response_content.get('data').get('links').get('job')
+response_content.get('data', {}).get('links', {}).get('job')
```

Reply:
This is dangerous, because if the response is not what expected, the job_url will be `None` and the polling will proceed to `<host>None` breaking things later on.